queries.sql

```diff
@@ -5,6 +5,47 @@ SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-0
 SELECT name FROM animals WHERE neutered IS true AND escape_attempts < 3;
 SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
-SELECT * FROM animals WHERE neutered IS true;
+SELECT * FROM animals WHERE neutered IS TRUE;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
-SELECT * FROM animals WHERE weight_kg >= 10.4  AND weight_kg <= 17.3;
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN WORK;
```

Comment:
- [ ] **[OPTIONAL]**  I suggest making use of just `BEGIN` instead of `BEGIN WORK` to begin a transition in all the places you have them because SQL Server does not support `BEGIN WORK`.